bip-0039/indonesian.txt

```diff
@@ -0,0 +1,2048 @@
+narkoba
+nasa
+nasional
+naskah
+natrium
+naungan
+navigasi
+nebula
+negara
+negeri
+negosiasi
+nelayan
+nenek
+neraka
+netral
+newspaper
+niat
+nilai
+ninja
+nirkabel
+nitrogen
+nominasi
+nomor
+nuansa
+nubuat
+nuklir
+nusa
+nutrisi
+nyaman
+nyanyian
+nyaris
+nyata
+nyawa
+nyeri
+obat
+objek
+obligasi
+oksigen
+olahraga
+oleh
+operasi
+opini
+oposisi
+opsi
+optik
+orang
+orbital
+ordo
+organisasi
+orientasi
+orkestra
+ornamen
+otak
+otomatis
+otonomi
+otoritas
+otot
+pabrik
+pacar
+pacu
+pada
+padi
+paduan
+pagar
+pagi
+paham
+pahit
+pahlawan
+pajak
+pakaian
+paket
+paksa
+paku
+paling
+palsu
+pamannya
+pameran
+panas
+pandangan
+panen
+panggung
+panitia
+panjang
+pantai
+papan
+para
+pariwisata
+parkir
+parlemen
+paroki
+partai
+paruh
+pasangan
+pasca
+pasir
+pasokan
+pasti
+pasukan
+patah
+paten
+patroli
+patung
+paus
+payung
+pecahan
+pecinta
+pedagang
+pedesaan
+pedoman
+peduli
+pegawai
+pegunungan
+pejabat
+pejuang
+pekan
+pekerjaan
+pelayanan
+pelepasan
+pelindung
+pelopor
+peluncuran
+pemain
+pembangunan
+pemerintah
+pemimpin
+pemrograman
+pemuda
+penampilan
+pencarian
+penduduk
+penerbangan
+pengguna
+peningkatan
+penjualan
+penonton
+pensiun
+penting
+penulis
+penyanyi
+peperangan
+pepohonan
+perang
+perbedaan
+percaya
+perdagangan
+perempuan
+perfilman
+pergi
+perhatian
+periode
+perjalanan
+perkembangan
+perlu
+permukaan
+pernah
+perolehan
+perpustakaan
+persen
+pertama
+perusahaan
+perwira
+pesawat
+peserta
+pesisir
+pesta
+petani
+peternakan
+peti
+petunjuk
+pewaris
+piala
+pidato
+pihak
+pikiran
+pilihan
+pimpinan
+pindah
+pinggiran
+pinjaman
+pintu
+piringan
+pisau
+pita
+plastik
+plat
+plot
+plus
+pohon
+poin
+pokok
+pola
+politik
+pondok
+ponsel
+populer
+poros
+posisi
+potensi
+potongan
+potret
+prajurit
+praktik
+pranala
+prasasti
+predator
+prefektur
+presiden
+pria
+pribadi
+primordial
+prinsip
+prioritas
+produksi
+profesional
+progresif
+promosi
+properti
+proses
+protein
+provinsi
+proyek
+psikologi
+puas
+publik
+pucat
+puing
+puisi
+pujian
+pukul
+pula
+pulih
+puluh
+punah
+puncak
+punggung
+punk
+punya
+pupuk
+pura
+purba
+pusat
+pustaka
+putaran
+putera
+putih
+putra
+putus
+racun
+radius
+ragam
+ragu
+rahasia
+raih
+raja
+rajin
+raksasa
+rakyat
+ramai
+rambut
+ranah
+rancangan
+rangka
+rantai
+rapat
+rapi
+rasa
+rasio
+rasul
+rating
+ratusan
+rawa
+raya
+reaksi
+realitas
+redaksi
+referensi
+reformasi
+regional
+reguler
+rekaman
+rekor
+rekreasi
+rektor
+relasi
+relevan
+religius
+remaja
+rempah
+renang
+rencana
+rendah
+renovasi
+rentang
+reproduksi
+republik
+reruntuhan
+resep
+resmi
+resolusi
+respon
+restoran
+revisi
+revolusi
+rezim
+ribu
+rilis
+rincian
+ringan
+rintisan
+riset
+risiko
+riwayat
+roda
+rohani
+roket
+rokok
+romantis
+rombongan
+ronde
+rongga
+rotasi
+roti
+ruang
+ruas
+rudal
+rugi
+rujukan
+rumah
+rumit
+rumput
+rumus
+runtuh
+rupa
+rupiah
+rusak
+rute
+rutin
+saat
+sabda
+sabuk
+sadar
+saham
+sains
+saja
+sakit
+saksi
+sakti
+salah
+saleh
+saling
+salju
+saluran
+sama
+sambil
+sampai
+samurai
```

Comment:
Imported word not familiar to most native speakers.

Reply:
Agreed on imported words, though I don't share your opinion of "samurai" being not familiar with most native speakers.